central/cluster/datastore/datastore_impl.go

```diff
@@ -918,10 +919,12 @@ func (ds *datastoreImpl) LookupOrCreateClusterFromConfig(ctx context.Context, cl
 			}
 		}
 
-		if cluster.GetInitBundleId() == bundleID &&
+		if !sensorCapabilitiesChanged(cluster, hello) &&
```

Comment:
This (commit 681852b248e35425ebe0ad2b4c7736968b62e068) is breaking the sensor upgrade test, and I don't know yet why.

I could also store the cert refresh capability in `ClusterStatus`:
```go
message ClusterStatus {
  string sensor_version = 1;
  // This field has been deprecated starting release 49.0. Use healthStatus.lastContact instead.
  google.protobuf.Timestamp DEPRECATED_last_contact = 2;
  ProviderMetadata provider_metadata = 3;
  OrchestratorMetadata orchestrator_metadata = 4;
  ClusterUpgradeStatus upgrade_status = 5;
  ClusterCertExpiryStatus cert_expiry_status = 6;
}
```

Since it fits well next to  `cert_expiry_status`.